tests/unit_test/related_articles/related_articles_config_test.py

```diff
@@ -0,0 +1,40 @@
+from data_pipeline.related_articles.related_articles_config import (
+    RelatedArticlesConfig
+)
+
+GIT_REPO_URL = 'git_repo_url'
+DIRECTORY_NAME = 'directory_name'
+
+SOURCE_CONFIG = {
+    'gitRepoUrl': GIT_REPO_URL,
+    'directoryName': DIRECTORY_NAME
+}
+
+PROJECT_NAME = 'project'
```

Comment:
Note: the reason why I usually add a number, like `_1`, is that the implementation could accidentally just return the config key `project` rather than the value from the dict. By using values other than the key, this would be more likely spotted.
(Just wanted to explain my reasoning behind it)